sdks/java/managed/src/main/java/org/apache/beam/sdk/managed/Managed.java

```diff
@@ -108,8 +107,8 @@ public static ManagedTransform read(String source) {
   }
 
   /**
-   * Instantiates a {@link Managed.Write} transform for the specified sink. The supported managed
-   * sinks are:
+   * Instantiates a {@link Managed.ManagedTransform} transform for the specified sink. The supported
```

Comment:
I think we wanted to introduced an outer "Managed" PTransform that will be returned here that may have Java-specific adapters etc. in the future.

Reply:
Can `ManagedTransform` take on that role? It extends SchemaTransform (ie. PTransform<PCollectionRowTuple, PCollectionRowTuple>) but we can change that to `PCollectionTuple` to include generic types in the future. Within, it can do whatever conversions necessary.